### PR TITLE
DT-355 for testing allow ITAG_USER to migrate keyworkers

### PR DESCRIPTION
**Pull request by Mjwillis**

### CODE REVIEW

File: src/main/resources/db/dev/data/nomis/V9_1__user_data.sql [hunk 1]
```diff
@@ -261,6 +261,7 @@ INSERT INTO USER_CASELOAD_ROLES (ROLE_ID, CASELOAD_ID, USERNAME)
 VALUES (-1, 'MDI', 'ITAG_USER'),
   (-1, 'MDI', 'OLD_NOMIS_USER'),
   (1, 'NWEB', 'ITAG_USER'),
+  (4, 'NWEB', 'ITAG_USER'),
```
Comment (JamieDraperUK) [inline src/main/resources/db/dev/data/nomis/V9_1__user_data.sql]: Are these meant to be unique?

Comment (brightonsbox) [reply src/main/resources/db/dev/data/nomis/V9_1__user_data.sql]: CONSTRAINT "USER_CASELOAD_ROLES_PK" PRIMARY KEY ("USERNAME", "CASELOAD_ID", "ROLE_ID")

So as long as the combo is unique this is fine

Comment (lbennett-moj) [reply src/main/resources/db/dev/data/nomis/V9_1__user_data.sql]: the role Id is different

Comment (Mjwillis) [reply src/main/resources/db/dev/data/nomis/V9_1__user_data.sql]: No, a user can have lots of roles

**Pull request merged**